crates/examples/testfiles/xcoff/base.o.objdump

```diff
@@ -8,15 +8,15 @@ Entry Address: 0
 1: Section { name: ".data", address: 68, size: 20, align: 4, kind: Data, flags: Xcoff { s_flags: 40 } }
 
 Symbols
-0: Symbol { name: ".file", address: 0, size: 0, kind: File, section: None, scope: Compilation, weak: false, flags: None }
-3: Symbol { name: ".printf", address: 0, size: 0, kind: Unknown, section: Undefined, scope: Unknown, weak: false, flags: None }
-5: Symbol { name: "printf", address: 0, size: 0, kind: Unknown, section: Undefined, scope: Unknown, weak: false, flags: None }
-7: Symbol { name: ".text", address: 0, size: 57, kind: Text, section: Section(SectionIndex(1)), scope: Dynamic, weak: false, flags: None }
-9: Symbol { name: ".main", address: 0, size: 0, kind: Text, section: Section(SectionIndex(1)), scope: Dynamic, weak: false, flags: None }
-11: Symbol { name: ".rodata.str1.1L...str", address: 58, size: d, kind: Text, section: Section(SectionIndex(1)), scope: Dynamic, weak: false, flags: None }
-13: Symbol { name: "main", address: 68, size: 18, kind: Data, section: Section(SectionIndex(2)), scope: Dynamic, weak: false, flags: None }
-15: Symbol { name: "TOC", address: 80, size: 0, kind: Data, section: Section(SectionIndex(2)), scope: Dynamic, weak: false, flags: None }
-17: Symbol { name: ".rodata.str1.1L...str", address: 80, size: 8, kind: Data, section: Section(SectionIndex(2)), scope: Dynamic, weak: false, flags: None }
+0: Symbol { name: ".file", address: 0, size: 0, kind: File, section: None, scope: Compilation, weak: false, flags: Xcoff { n_sclass: 67 } }
+3: Symbol { name: ".printf", address: 0, size: 0, kind: Section, section: Undefined, scope: Unknown, weak: false, flags: Xcoff { n_sclass: 2 } }
+5: Symbol { name: "printf", address: 0, size: 0, kind: Section, section: Undefined, scope: Unknown, weak: false, flags: Xcoff { n_sclass: 2 } }
+7: Symbol { name: ".text", address: 0, size: 57, kind: Section, section: Section(SectionIndex(1)), scope: Dynamic, weak: false, flags: Xcoff { n_sclass: 6b } }
+9: Symbol { name: ".main", address: 0, size: 0, kind: Section, section: Section(SectionIndex(1)), scope: Dynamic, weak: false, flags: Xcoff { n_sclass: 2 } }
+11: Symbol { name: ".rodata.str1.1L...str", address: 58, size: d, kind: Section, section: Section(SectionIndex(1)), scope: Dynamic, weak: false, flags: Xcoff { n_sclass: 6b } }
+13: Symbol { name: "main", address: 68, size: 18, kind: Section, section: Section(SectionIndex(2)), scope: Dynamic, weak: false, flags: Xcoff { n_sclass: 2 } }
+15: Symbol { name: "TOC", address: 80, size: 0, kind: Section, section: Section(SectionIndex(2)), scope: Dynamic, weak: false, flags: Xcoff { n_sclass: 6b } }
+17: Symbol { name: ".rodata.str1.1L...str", address: 80, size: 8, kind: Section, section: Section(SectionIndex(2)), scope: Dynamic, weak: false, flags: Xcoff { n_sclass: 6b } }
```

Comment:
Changing all of the symbols in the test results to `kind: Section` can't possibly be correct.